app/config/swap/testnet.json

```diff
@@ -0,0 +1,7 @@
+{
+	"packageId": "0x4995e309e990a6a93224153108b26bf79197b234c51db6447bbae10b431c42fb",
+	"module": "nbtc_swap",
+	"swapFunction": "swap_sui_for_nbtc",
+	"vaultId": "0xf280477ca196a4bced5e1db4cd82fcdd647b55585b1d3838dcd8e1b829d263a4",
+	"pricePerNBTCInSUI": 25000
```

Comment:
this is not part of network config. Maybe we should keep is separately

Reply:
let's keep it - but in the different structure. I made a comment above.

Reply:
oh, I see what you mean - maybe you are right. we can have network config and contracts:
- config/sui/network-testnet.json
- config/sui/network-mainnet.json
- config/sui/contracts-testnet.json
- ...

Reply:
Done